src/controller/checkout.controller.ts

```diff
@@ -21,7 +21,7 @@ export default function (usersService: UsersService, paymentsService: PaymentSer
       timeWindow: '1 minute',
     });
 
-    fastify.addHook('onRequest', async (request, reply) => {
+    fastify.addHook('onRequest', async (request, _) => {
```

Comment:
If you are not using the `reply`, just do not declare it on the function (aka remove the '_')